src/obb/impl/main.cljs

```diff
@@ -47,16 +52,35 @@
   [s]
   (sci/eval-string* ctx s))
 
+(def print*
+  ;; All output from osascript by default goes to stderr. To get around this
+  ;; we use the Objective-C bridge to write directly to stdout.
+  (let [import (delay (.import js/ObjC "Foundation"))]
+    (fn [s]
+      @import
+      (-> (.dataUsingEncoding (js/$.NSString.alloc.initWithString (str s))
+                              js/$.NSUTF8StringEncoding)
+          (js/$.NSFileHandle.fileHandleWithStandardOutput.writeData)))))
```

Comment:
I'm not sure how this works, but is there a concern of freeing/closing things after printing?

Reply:
Yeah, I'm very new to Objective-C so there could easily be several things that are subtly wrong with this. It's taken more or less directly from an example in the JXA cookbook. I haven't seen people releasing `NSString` instances, but that could be because in the scripting environment they expect the entire program to be short-lived. 

I'll dig into it further.

Reply:
I looked around and I wasn't able to find any examples of people explicitly releasing memory using JXA. That said, this still makes me nervous so I'm going to replace the call to `$.NSString.alloc` with a call to `ObjC.wrap`. That should hopefully address any memory management issues.